client/apps/city_issues/user_managers.py

```diff
@@ -18,7 +18,7 @@ def _create_user(self, name, email, password, **extra_fields):
         user.save(using=self._db)

```

Comment:
why we use "using=self._db". we use only one database currently

Reply:
yes, we use just 1 database and simply saving data doesn't create a new database

Reply:
it's just a custom user saving method :)

Reply:
https://github.com/django/django/blob/master/django/contrib/auth/models.py